agent/stats/common_test.go

```diff
@@ -154,6 +154,9 @@ func validateContainerMetrics(containerMetrics []*ecstcs.ContainerMetric, expect
 		if containerMetric.MemoryStatsSet == nil {
 			return fmt.Errorf("MemoryStatsSet is nil")
 		}
+		if containerMetric.NetworkStatsSet == nil {
+			return fmt.Errorf("NetworkStatsSet is nil")
+		}
```

Comment:
Should we also check the type of the stats set here or is that overkill?
In my additions for `storageReadStats` and storageWriteStats` in the `createFakeContainerStats()` function below, I just dropped in a uint64(0) for each; this will pass the null checks above.

Reply:
> Should we also check the type of the NetworkStatsSet here? 

what does this mean?

Reply:
a type check like:
```
networkType, ok := containerMetric.NetworkStatsSet.(ecstcs.NetworkStatsSet)
if !ok {
    return fmt.Errorf("NetworkStatsSet is invalid type")
}
```
Like I said... might be overkill.